### ISSUE TITLE
Build error 'cannot read property startsWith of undefined' when using adaptive icons

### ISSUE THREAD

**Issue by AshleyScirra** [apache/cordova-android#984]
# Bug Report

Sample project: 
[test.zip](https://github.com/apache/cordova-cli/files/4703578/test.zip)

Extract and run:
`cordova platform add android`
`cordova prepare android`

## Problem

It fails with `Cannot read property 'startsWith' of undefined`.

### What is expected to happen?

Successful prepare

### What does actually happen?

It fails with an error message

## Information

This only happens if you have both a normal and an adaptive icon.

### Command or Code
Included above

### Environment, Platform, Device
Windows 10 64-bit

### Version information
Cordova 9.0.0 (cordova-lib@9.0.1)

## Checklist

- [x] I searched for existing GitHub issues
- [x] I updated all Cordova tooling to most recent version
- [x] I included all the necessary information above

Note: I wasn't sure where the right place to file this was, so it's also filed at https://github.com/apache/cordova-cli/issues/506

**Comment by breautek** [comment]
Observed the error on nightly builds.

**Comment by breautek** [comment]
If you can test #1001 PR to ensure that the PR addresses your issue, that would be greatly appreciated :+1:

**Comment by AshleyScirra** [comment]
I'd like to test it, but it's not clear how to apply that patch to an existing Cordova setup. I can't find the `updateIconResourceForAdaptive` method. Are there nightly releases of Cordova I could try? Or is there any indication when the next Android release will be? It looks like the last release was several months ago.

**Comment by breautek** [comment]
It's already merged in master, so you can install from git master...

```
cordova platform remove android
cordova platform add https://github.com/apache/cordova-android.git
```

It will appear on the next nightly build, but that's in like 10-11 hours from now.

If you could verify asap that would be helpful, because we are *planning* on starting the release process very soon.

**Comment by AshleyScirra** [comment]
Using `cordova platform add https://github.com/apache/cordova-android.git` and `cordova prepare` appears to succeed without the error. I happened to run it in WSL if it makes any difference, that also reproduced the original issue only using `cordova platform add android` (i.e. latest published). So it looks like it's fixed.

**Comment by breautek** [comment]
Great! Thank you for your time and effort, and of course bringing attention to this issue to us. The fix will be included in the 9.0.0 release, which should be happening soon.

**Comment by AshleyScirra** [comment]
Good to know, thanks!
